BlazarUI/app/scripts/components/branch/BranchHeadline.jsx

```diff
@@ -28,6 +29,9 @@ class RepoBuildHeadline extends Component {
         <HeadlineDetail>
           Branch Builds
         </HeadlineDetail>
+        <HeadlineDetail>
+          <Image src={imgPath}/>
```

Comment:
You can put this `Image` in the other `HeadlineDetail` instead of creating two instances:

```
<HeadlineDetail>
  Branch Builds
  <Image src={imgPath} />
</HeadlineDetail>
```